src/test/resources/application-test-c100-validation.yaml

```diff
@@ -0,0 +1,130 @@
+uk:
+  gov:
+    hmcts:
+      form-validation:
+        caseTypeofApplication:
+          C100:
+            mandatoryFields:
+              - previous_or_ongoingProceeding
+              - existingCase_onEmergencyProtection_Care_or_supervisioNorder
+              - exemption_to_attend_MIAM
+              - familyMember_Intimation_on_No_MIAM
+              - attended_MIAM
+              - NoMIAM_domesticViolence
+              - NoMIAM_childProtectionConcerns
+              - NoMIAM_Urgency
+              - NoMIAM_PreviousAttendance
```

Comment:
Need to remove this file all places

Reply:
Done. Conditional mandatory fields now moved to Optional field group.